cmd/kubeadm/app/phases/token/bootstrap.go

```diff
@@ -36,18 +36,18 @@ const (
 
 // UpdateOrCreateToken attempts to update a token with the given ID, or create if it does
 // not already exist.
-func UpdateOrCreateToken(client *clientset.Clientset, d *kubeadmapi.TokenDiscovery, tokenDuration time.Duration) error {
+func UpdateOrCreateToken(client *clientset.Clientset, d *kubeadmapi.TokenDiscovery, tokenDuration time.Duration, usages []string) error {
 	// Let's make sure the token is valid
 	if valid, err := tokenutil.ValidateToken(d); !valid {
 		return err
 	}
-	secretName := fmt.Sprintf("%s%s", kubeadmconstants.BootstrapTokenSecretPrefix, d.ID)
+	secretName := kubeadmconstants.BootstrapTokenSecretPrefix + d.ID
```

Comment:
Same thing here -- we should really make this look at all secrets vs making assumptions about name.